.github/workflows/linux.yml

```diff
@@ -302,7 +302,7 @@ jobs:
 
   Conformance:
     needs: [ Build, Smart_CI ]
-    timeout-minutes: ${{ matrix.TEST_TYPE == 'API' && 5 || 15 }}
+    timeout-minutes: ${{ matrix.TEST_TYPE == 'API' && 5 || matrix.TEST_TYPE == 'OP' && 20 }}
```

Comment:
no need to add `matrix.TEST_TYPE == 'OP'`, just changing to `20` should be enough as there are only two configs: `API`/`OP`.